scripts/jest/jest-resources.js

```diff
@@ -73,8 +73,6 @@ module.exports = {
 
         globals: {
           'ts-jest': {
-            tsConfig: path.resolve(packageRoot, 'tsconfig.json'),
```

Comment:
deprecated ts-jest settings

Reply:
this will need to be removed in all vNext packages ( out of scope this PR )

Reply:
https://github.com/microsoft/fluentui/issues/20826

Reply:
I'm not seeing anything about this being deprecated, just that it's been renamed from `tsConfig` to `tsconfig`. https://github.com/kulshekhar/ts-jest/blob/main/CHANGELOG.md